client/src/components/mapping/FhirMappingPanel/FhirResourceTree/index.tsx

```diff
@@ -251,8 +262,19 @@ const FhirResourceTree = ({ onClickCallback }: Props) => {
     return node;
   };
 
+  // For source readers, we only want to display attributes that are filled
+  const shoudlBeDisplayed = (pathString: string) =>
+    isAdmin ||
+    isWriter ||
+    attributesWithInputs.some(
+      el => el !== pathString && el.startsWith(pathString)
+    );
+
```

Comment:
@elsiehoffet-94  @nriss @Peltier10  do we want to make this switchable ? would it be useful for an ADMIN to benefit form this feature (only see filled attributes) ?

Reply:
> @elsiehoffet-94 @nriss @Peltier10 do we want to make this switchable ? would it be useful for an ADMIN to benefit form this feature (only see filled attributes) ?

For now, I'm not sure I will use that kind of functionality as a mapper.
Personally, I prefer seeing all available attributes

Reply:
- I think it's an enjoyable option for an admin or writer user to only see filled attributes for a final check of our mapping. 

- I agree with @nriss on the second point, it would be nice to develop the tree on click for a terminal attribute, but this might be complicated for complex arrays, with multiple attributes of type CodeableConcepts. I think it would be prefereable to have this option only on filled attributes. And it's a feature request, and can be the object of a different issue.

Reply:
I agree with @elsiehoffet-94 and @nriss. It can be useful to make a last check up of the mapping.